pyproject.toml

```diff
@@ -1,4 +1,107 @@
+[tool.poetry]
+name = "dowhy"
+version = "0.8.0"
```

Comment:
I recently introduced versioneer for versioning. That means all versioning information comes from git tags. So we need to remove explicit versioning in text files. Can you check how this is compatible with Poetry?

Reply:
It looks like there's a plugin that will enable versioning based on git tags: https://pypi.org/project/poetry-dynamic-versioning/

Reply:
I've add some configuration for `poetry-dynamic-versioning`; but there don't appear to be any git tags in my clone. You may want to test this out in your checkout. 

The plugin will perform text replacement of the version string in `pyproject.toml` and `dowhy/__init__.py` - it expects to replace "0.0.0"